src/main/webapp/pages/UserPage.html

```diff
@@ -2,24 +2,31 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Capstone</title>
+    <title>User Page</title>
     <link rel="stylesheet" href="../css/style.css">
     <link rel="stylesheet" href="../css/style-userpage.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <script src="../javascript/user-page.js"></script>
+    <script src="../javascript/RecipeRendering.js"></script>
+    <script src="https://kit.fontawesome.com/9d7d4957c5.js" crossorigin="anonymous"></script>
   </head>
   <body onload="loadRecipes()">
     <div id="content">
       <h1 class="center">My Page</h1>
       <!-- Planner content -->
       <div>
-        <h2>Planner</h2>
-        <div class="recipes-container">
+        <div id="header">
+          <h2>Planner</h2>
+        </div>
+        <div class="recipes-container curve" id="planner-recipes">
         </div>
       </div>
       <!-- Cookbook content -->
       <div>
-        <h2>Cookbook</h3>
-        <div class="recipes-container">
+        <div id="header">
+          <h2>Cookbook</h3>
```

Comment:
should open tag and close tag both be h2?

Reply:
lol yeah